packages/components/theme.ts

```diff
@@ -8,49 +8,32 @@ export const theme = {
         heading: 'IBM Plex Sans, sans-serif',
     },
     buttons: {
-        uno: {
+        primary: {
             cursor: 'pointer',
-            bg: 'gray.3',
-            color: 'gray.8',
-            borderRadius: 'sm',
+            bg: 'rgba(16, 16, 16, 1)',
+            color: '#fff',
+            borderRadius: '36px',
+            paddingTop: '2px',
+            paddingBottom: '2px',
             px: 4,
-            ':hover': {
-                bg: 'gray.7',
-                color: 'gray.3',
-            },
         },
-        due: {
+        secondary: {
             cursor: 'pointer',
-            bg: 'gray.8',
-            color: 'gray.3',
-            borderRadius: 'sm',
+            bg: '#fff',
+            color: 'rgba(16, 16, 16, 1)',
+            border: '1px solid rgba(16, 16, 16, 1)',
+            borderRadius: '36px',
+            paddingTop: '2px',
```

Comment:
You can use `pt` instead of `paddingTop` and `pb` instead of `paddingBottom` and `py` to define top and bottom paddign at the same time